fractions/monitor/src/main/java/org/wildfly/swarm/monitor/runtime/InstallMonitorFilter.java

```diff
@@ -47,10 +42,11 @@
 
     @Override
     public void prepareArchive(Archive<?> archive) {
-        JARArchive jarArchive = archive.as(JARArchive.class);
-
-        Asset resource = new ClassAsset(HealthResponseFilter.class);
-        ArchivePath location = new BasicPath(PATH_CLASSES, AssetUtil.getFullPathForClassResource(HealthResponseFilter.class));
-        jarArchive.add(resource, location);
+        try {
```

Comment:
yes, but for some reason it didn't resolve the artefact without a specific version